src/runtime/graph/graph_runtime.cc

```diff
@@ -53,7 +53,15 @@ inline size_t GetDataAlignment(const DLTensor& arr) {
 void GraphRuntime::Run() {
   // setup the array and requirements.
   for (size_t i = 0; i < op_execs_.size(); ++i) {
-    if (op_execs_[i]) op_execs_[i]();
+    if (op_execs_[i]) {
+      auto& op_arg = op_args_[i];
```

Comment:
@yongsun @icemelon9 where is this op_args_? there is only op_args in the code, and my build failed because op_args_ cannot be found. How does this even get through CI?